server/routes/overview/overviewController.ts

```diff
@@ -117,22 +110,11 @@ export default class OverviewController {
     const allFunctionalSkills = await this.curiousService.getPrisonerFunctionalSkills(prisonNumber, req.user.username)
     const functionalSkills = mostRecentFunctionalSkills(allFunctionalSkills)
 
-    const allInPrisonEducation = await this.curiousService.getLearnerEducation(prisonNumber, req.user.username)
-    const completedInPrisonEducation = completedInPrisonEducationRecords(allInPrisonEducation)
-    const completedInPrisonEducationWithinLast12Months = await this.setPrisonNamesOnEducationRecords(
-      completedInPrisonEducationRecordsWithinLast12Months(completedInPrisonEducation),
-      req,
-    )
+    const inPrisonCourses = await this.curiousService.getPrisonerInPrisonCourses(prisonNumber, req.user.username)
```

Comment:
The controller code is much simpler now, now that it doesnt have to faff around with filtering and mutating the course data it gets from the service 👍

Reply:
And my next PR, which is waiting in the wings for this one to be merged first, will make the controller code even simpler still in respect of getting data like this from services 👍  (bet you can't wait! 🤣 )